x-pack/platform/plugins/shared/fleet/public/applications/fleet/sections/settings/components/edit_output_flyout/output_form_secret_form_row.tsx

```diff
@@ -23,6 +23,13 @@ import React, { useMemo, useState } from 'react';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { i18n } from '@kbn/i18n';
 
+import {
+  SSL_SECRETS_MINIMUM_FLEET_SERVER_VERSION,
+  OUTPUT_SECRETS_MINIMUM_FLEET_SERVER_VERSION,
+} from '../../../../../../../common/constants';
+
+export type SecretType = 'output' | 'ssl';
```

Comment:
I thought that the output secrets were supported since long time. What is the case for outputs that need the new version of fleet server?

Reply:
that's true, we can probably leave the secret that ends up in the output in the previous version check

Reply:
updated, had to refactor to handle the separate ssl and output secret enabled state in the fleet server host form
https://github.com/elastic/kibana/pull/237464/commits/e3784193ae3222aaced78ceed721e9649965f343